src/qt/sendcoinsdialog.cpp

```diff
@@ -318,11 +318,24 @@ void SendCoinsDialog::on_sendButton_clicked()
         // append fee string if a fee is required
         questionString.append("<hr /><span style='color:#aa0000;'>");
         questionString.append(BitcoinUnits::formatHtmlWithUnit(model->getOptionsModel()->getDisplayUnit(), txFee));
+
```

Comment:
Is there a reason for this newline and the double-newline in L333?

Reply:
Fixed in e8485d10e63c1c66ea85b253ff38ee86fd2a3ea6